Source/CesiumRuntime/Private/CesiumGltfComponent.cpp

```diff
@@ -1210,10 +1216,29 @@ static void loadPrimitive(
   primitiveResult.pModel = &model;
   primitiveResult.pMeshPrimitive = &primitive;
   primitiveResult.RenderData = std::move(RenderData);
-  primitiveResult.transform = transform;
   primitiveResult.pMaterial = &material;
   primitiveResult.pCollisionMesh = nullptr;
 
+  static constexpr glm::dmat4 scaleMatrix = {
```

Comment:
Let's give this a better name and a comment explaining why it exists. It's not scaling, just inverting the Y-axis.